lib/4_filter_numbers.js

```diff
@@ -6,6 +6,8 @@
 
 module.exports = {
   filter_numbers: function(arr){
-    // Your code goes here.
+  var lessThan10 = arr.filter(function (x){
```

Comment:
Within the outermost function, spacing your work like so makes it easier to parse:

```javascript
var lessThanTen = arr.filter(function(x) {
  return x < 10;
});
return lessThanTen;